pkg/constant/constant_test.go

```diff
@@ -132,25 +132,9 @@ func TestContainerdModuleVersions(t *testing.T) {
 
 func TestRuncModuleVersions(t *testing.T) {
```

Comment:
The runc dependency is gone, I kept this to just check that vars.sh returns a runc_version number that makes sense. I'm not too sure what to do about this, maybe we should entirely remove the test or leave it as is because it's not hurting anyway...

Reply:
Oh, this is already the second time the dependency vanished from upstream. I'd be comfortable with removing the test entirely. I think that's what I did last time (need to check the git history).

Reply:
Removed the test entirely.